drivers/sensor/ina23x/ina237.h

```diff
@@ -43,6 +43,7 @@ struct ina237_data {
 	uint16_t bus_voltage;
 	uint32_t power;
 	int16_t die_temp;
+	int16_t shunt_voltage;
```

Comment:
This can be ifdef protected, so it takes take space in the struct unless enabled.

Reply:
Good point, done and pushed.